docs/developer_guide/release_process.md

```diff
@@ -108,7 +108,7 @@ If there are any changes, commit them.
 
 Create a PR.
 
-## 7. For major releases, create an release-X.Y branch
+## 7. For major releases, create n release-X.Y branch
```

Comment:
a?

Reply:
Sorry!  Thank you for the catch!

Reply:
@arnavgarg1 Now please -- fixed!  Thanks!